crates/matrix-sdk-crypto-js/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+authors = ["Ivan Enderlin <ivane@element.io>"]
+description = "Matrix encryption library"
+edition = "2021"
+homepage = "https://github.com/matrix-org/matrix-rust-sdk"
+keywords = ["matrix", "chat", "messaging", "ruma", "nio"]
+license = "Apache-2.0"
+name = "matrix-sdk-crypto-js"
+readme = "README.md"
+repository = "https://github.com/matrix-org/matrix-rust-sdk"
+rust-version = "1.60"
+version = "0.5.0"
+
+[package.metadata.docs.rs]
+features = ["docsrs"]
+rustdoc-args = ["--cfg", "docsrs"]
+
+[package.metadata.wasm-pack.profile.release]
+wasm-opt = ['-Oz']
+
+[lib]
+crate-type = ["cdylib"]
+
+[features]
+default = ["js"]
+qrcode = ["matrix-sdk-crypto/qrcode"]
+backups_v1 = ["matrix-sdk-crypto/backups_v1"]
+docsrs = []
```

Comment:
Not sure they are required.

Reply:
The `qrcode` feature yes. The `backups_v1` feature, yes but that pulls in `olm-sys` and `olm-rs` AKA the C libolm cryptographic library. So this wouldn't work for WASM. Ignore `backups_v1` for now.

Reply:
Solved.